tests/unit/api/bucketGet.js

```diff
@@ -108,6 +107,25 @@ const tests = [
             assert.strictEqual(result.ListBucketResult.Contents[1], undefined);
         },
     },
+    {
+        name: 'next token is not url encoded',
+        request: Object.assign(
+            {
+                query: { 'encoding-type': 'url', 'max-keys': '1' },
+                url: baseUrl,
+            },
+            baseGetRequest
+        ),
+        assertion: result => {
+            assert.strictEqual(result.ListBucketResult.Contents[0].Key[0],
+                objectName3);
+            assert.strictEqual(result.ListBucketResult.Contents[1], undefined);
+            assert.strictEqual(
+                result.ListBucketResult.NextContinuationToken[0],
+                'aW52YWxpZFVSSX5+fmI='
```

Comment:
would this be the value every time?

Reply:
the value is the base64 encoding of objectName3